src/loaders/svg/tvgSvgLoader.h

```diff
@@ -31,6 +31,7 @@ class SvgLoader : public LoadModule, public Task
     string filePath;
     const char* content = nullptr;
     uint32_t size = 0;
+    float width = 0.0f, height = 0.0f; //svg tag attributes
```

Comment:
I kept for now (or rather moved from the LoadModule) 'width' and 'height' members. 
ThorVG doesn't support scaling of the image with these values. Maybe after we fix it, we decide to remove these two.